bedrock/firefox/tests/test_views.py

```diff
@@ -634,6 +634,38 @@ def test_firefox_home(self, render_mock):
         assert template == ["firefox/home/index-master.html"]
 
 
+class TestFirefoxGA(TestCase):
+    def assert_ga_attr(self, response):
+        cta_types = ["link", "button"]
```

Comment:
Suggestion (non-blocking): another value that seems common is `fxa-*` (e.g. `fxa-sync` `fxa-monitor` etc). They don't seem to be needed for these particular tests to pass, but if we wanted to expand these tests out to cover other pages then they might be worth including. Up to you here 👍

Reply:
yeah! i added that below on line 647: 
```python
  if cta_data:
      contain_cta = any(cta_data in list for list in cta_types)
      assert contain_cta or "fxa-" in cta_data
```